src/renameAll.js

```diff
@@ -0,0 +1,13 @@
+const apply   = require('ramda/src/apply')
+const compose = require('ramda/src/compose')
+const curry   = require('ramda/src/curry')
+const map     = require('ramda/src/map')
+const toPairs = require('ramda/src/toPairs')
+
+const rename = require('./rename')
+
+// renameAll :: { k: v } -> { k: v } -> { k: v }
+const renameAll = (renames, obj) =>
```

Comment:
am I crazy or could this just be:

```javascript
const renameAll =
  compose(apply(compose), map(apply(rename)), toPairs)

module.exports = curry(renameAll)
```

Reply:
Would need crocks.curry for that to work.

Reply:
Only if we pulled in `crocks#curry`.  Ramda's wouldn't do it right, and we haven't made `crocks` a dep of this library yet.